acl_loader/main.py

```diff
@@ -453,6 +481,41 @@ def convert_ip(self, table_name, rule_idx, rule):
 
         return rule_props
 
+    def convert_icmp(self, table_name, rule_idx, rule):
+        rule_props = {}
+
+        is_table_v6 = self.is_table_ipv6(table_name)
+        type_key = "ICMPV6_TYPE" if is_table_v6 else "ICMP_TYPE"
+        code_key = "ICMPV6_CODE" if is_table_v6 else "ICMP_CODE"
+
+        if rule.icmp.config.type or rule.icmp.config.type == 0:
+            try:
+                icmp_type = int(rule.icmp.config.type)
```

Comment:
>rule.icmp.config.type [](start = 32, length = 21)

Is it possible rule.icmp.config.type is an integer? if yes, you don't need cast to int.